packages/wallet/src/redux/protocols/consensus-update/__tests__/reducer.test.ts

```diff
@@ -9,7 +9,7 @@ import {
 
 import * as scenarios from "./scenarios";
 
-describe("Two Players", () => {
+describe.skip("Two Players", () => {
```

Comment:
Outside the scope of this PR but I think it might make sense to remove the `wallet` package now.